src/Api/Struct/Webspace/DiskUsage.php

```diff
@@ -0,0 +1,66 @@
+<?php
+// Copyright 1999-2019. Plesk International GmbH.
+// Author: Frederic Leclercq
+
+namespace PleskX\Api\Struct\Webspace;
+
+class DiskUsage extends \PleskX\Api\Struct
+{
+    /** @var string */
+    public $name;
+
```

Comment:
Seems name and guid not returned for disk_usage according to reference
https://docs.plesk.com/en-US/onyx/api-rpc/about-xml-api/reference/managing-sites-domains/getting-information-about-sites.66583/
Lets remove this